molgenis-navigator/src/main/frontend/test/unit/specs/store/actions.spec.js

```diff
@@ -0,0 +1,330 @@
+import api from '@molgenis/molgenis-api-client'
+import td from 'testdouble'
+import actions, { GET_ENTITIES_IN_PACKAGE } from 'src/store/actions'
+import { RESET_PATH, SET_ENTITIES, SET_ERROR, SET_PACKAGES, SET_PATH } from 'src/store/mutations'
+import testAction from '../../utils/action.utils'
+
+describe('actions', () => {
+  afterEach(() => td.reset())
+
+  describe('QUERY_PACKAGES', function () {
+    it('should fetch the packages and call the SET_PACKAGES mutation', done => {
+      const package1 = {id: 'pack1', label: 'packLabel1'}
+      const response = {
+        items: [package1]
+      }
+
+      const get = td.function('api.get')
+      td.when(get('/api/v2/sys_md_Package?sort=label&num=1000')).thenResolve(response)
```

Comment:
I really like the fact that the exact query string is checked in this test solution.